### DISCUSSION TITLE
There is error when run  ./update_and_rebuild_petsc.sh

### DISCUSSION THREAD

**Discussion by liuyong201811** [idaholab/moose#19378] [Q&A General] [answered]
When  I run ./update_and_rebuild_petsc.sh
Cannot use fblaslapack without Fortran, make sure you do NOT have --with-fc=0
*********
Thank you.

![moose](https://user-images.githubusercontent.com/44690649/141644082-9e59c499-295f-4d02-ae2b-f74805af94c2.png)
**********************************************************************

There was an error. Exiting...
![moose](https://user-images.githubusercontent.com/44690649/141644076-1f91be02-35a8-48ff-a90f-1091696e087d.png)

TESTING: alternateConfigureLibrary from config.packages.szlib(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.zlib(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.zlib(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.tetgen(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.tetgen(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.tchem(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.tchem(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.saws(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.saws(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.libpng(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.libpng(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.combblas(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.combblas(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.Triangle(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.Triangle(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.metis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: checkDependencies from config.packages.metis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:872)
TESTING: configureLibrary from config.packages.metis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/packages/metis.py:46)
                        =============================================================================================
                              Trying to download git://https://bitbucket.org/petsc/pkg-metis.git for METIS
                        =============================================================================================

                        =============================================================================================
                              Configuring METIS with cmake; this may take several minutes
                        =============================================================================================

                        =============================================================================================
                              Compiling and installing METIS; this may take several minutes
                        =============================================================================================

TESTING: check from config.libraries(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/libraries.py:157)
TESTING: checkInclude from config.headers(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/headers.py:86)
TESTING: checkInclude from config.headers(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/headers.py:86)
TESTING: checkVersion from config.packages.metis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:1026)
TESTING: checkSharedLibrary from config.packages.metis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:955)
TESTING: consistencyChecks from config.packages.pragmatic(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.pragmatic(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.parmetis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: checkDependencies from config.packages.parmetis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:872)
TESTING: configureLibrary from config.packages.parmetis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/packages/parmetis.py:41)
                        =============================================================================================
                              Trying to download git://https://bitbucket.org/petsc/pkg-parmetis.git for PARMETIS
                        =============================================================================================

                        =============================================================================================
                              Configuring PARMETIS with cmake; this may take several minutes
                        =============================================================================================

                        =============================================================================================
                              Compiling and installing PARMETIS; this may take several minutes
                        =============================================================================================

TESTING: check from config.libraries(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/libraries.py:157)
TESTING: checkInclude from config.headers(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/headers.py:86)
TESTING: checkInclude from config.headers(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/headers.py:86)
TESTING: checkVersion from config.packages.parmetis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:1026)
TESTING: checkSharedLibrary from config.packages.parmetis(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:955)
TESTING: consistencyChecks from config.packages.Chaco(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.Chaco(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.hdf5(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.hdf5(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.med(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.med(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.cgns(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.cgns(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.ascem-io(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.ascem-io(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.pflotran(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.pflotran(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.alquimia(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.alquimia(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.SAMRAI(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.SAMRAI(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.ADIOS2(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.ADIOS2(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.pnetcdf(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.pnetcdf(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.netcdf(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.netcdf(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.exodusii(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
TESTING: alternateConfigureLibrary from config.packages.exodusii(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:959)
TESTING: consistencyChecks from config.packages.fblaslapack(/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py:963)
*******************************************************************************
         UNABLE to CONFIGURE with GIVEN OPTIONS    (see configure.log for details):
-------------------------------------------------------------------------------
Cannot use fblaslapack without Fortran, make sure you do NOT have --with-fc=0
*******************************************************************************

There was an error. Exiting...

**Comment by GiudGiud** [comment]
Hello

What compilers are you using? It says earlier in the log I think.
You can also query:
```
echo $CC $CXX $FC
```
then for good measure please report their version too

Guillaume

**Reply by GiudGiud** [reply]
@liuyong201811 what does this return?

**Reply by liuyong201811** [reply]
mpicc mpicxx mpif90

**Reply by GiudGiud** [reply]
and mpicc --version.
I m expecting clang 10.0, but just checking

**Reply by GiudGiud** [reply]
ok checked the configure log and it confirms clang 10 is being used. There s still some references to the installed gcc version I find suspicious but maybe that's normal, like this one `Selected GCC installation: /usr/lib/gcc/aarch64-redhat-linux/4.8.5`

You are getting a bunch of fortran and BLAS errors at the end. Have you tried the solution they suggest in the script?

**Reply by liuyong201811** [reply]
mpicc --version
Bisheng Compiler 1.3.3.b023 clang version 10.0.1 (clang-e31092d4f8cd flang-3c92ea4b404f)
Target: aarch64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/compiler/bisheng-compiler-1.3.3-aarch64-linux/bin

**Reply by liuyong201811** [reply]
They did not give the method to solution a bunch of fortran and BLAS errors at the end. Or they give but I do not know to do it.thank you

**Reply by GiudGiud** [reply]
~~open the `update_and_rebuild_petsc.sh` script in a text editor and change `--download-fblaslapack=0` to `--download-fblaslapack=1`~~
see below.

**Reply by loganharbour** [reply]
You can just do:
```
./scripts/update_and_rebuild_petsc.sh --download-fblaslapack=1
```

Arguments are passed to petsc configure.

**Reply by liuyong201811** [reply]
I try it but it did work.
then the wrong changed into 
*******************************************************************************
         UNABLE to CONFIGURE with GIVEN OPTIONS    (see configure.log for details):
-------------------------------------------------------------------------------
Error configuring STRUMPACK with cmake
*******************************************************************************
  File "/path/to/MOOSE/moose/petsc/config/configure.py", line 458, in petsc_configure
    framework.configure(out = sys.stdout)
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/framework.py", line 1312, in configure
    self.processChildren()
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/framework.py", line 1300, in processChildren
    self.serialEvaluation(self.childGraph)
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/framework.py", line 1275, in serialEvaluation
    child.configure()
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 1146, in configure
    self.executeTest(self.configureLibrary)
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/base.py", line 138, in executeTest
    ret = test(*args,**kargs)
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 904, in configureLibrary
    for location, directory, lib, incl in self.generateGuesses():
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 478, in generateGuesses
    d = self.checkDownload()
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 612, in checkDownload
    return self.getInstallDir()
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 367, in getInstallDir
    installDir = self.Install()
  File "/path/to/MOOSE/moose/petsc/config/BuildSystem/config/package.py", line 1792, in Install
    raise RuntimeError('Error configuring '+self.PACKAGE+' with cmake')
================================================================================
Finishing configure run at Mon, 15 Nov 2021 11:39:30 +0800
================================================================================

**Comment by liuyong201811** [comment]
The wrong changed into 
         UNABLE to CONFIGURE with GIVEN OPTIONS    (see configure.log for details):
-------------------------------------------------------------------------------
Could not find a functional BLAS. Run with --with-blas-lib=<lib> to indicate the library containing BLAS.
 Or --download-fblaslapack=1 to have one automatically downloaded and installed
Where can I set --with-blas-lib or --download-fblaslapack=1, thank you!

**Reply by GiudGiud** [reply]
you can change those options inside the update_and_rebuild_petsc.sh script. 
This is kind of an advanced use case though. There is likely another issue

**Reply by liuyong201811** [reply]
[configure.log](https://github.com/idaholab/moose/files/7533299/configure.log)
this is the configure.log, can you help me to see what is the issue,thank you.

**Reply by GiudGiud** [reply]
Is your gcc version 4.8?
Minimum required is 5.1
https://mooseframework.inl.gov/getting_started/installation/manual_installation_gcc.html

**Comment by liuyong201811** [comment]
I use the clang to compile the program .

[root@DC5-02-010 moose]# clang -v
Bisheng Compiler 1.3.3.b023 clang version 10.0.1 (clang-e31092d4f8cd flang-3c92ea4b404f)
Target: aarch64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/compiler/bisheng-compiler-1.3.3-aarch64-linux/bin
Found candidate GCC installation: /usr/lib/gcc/aarch64-redhat-linux/4.8.2
Found candidate GCC installation: /usr/lib/gcc/aarch64-redhat-linux/4.8.5
Selected GCC installation: /usr/lib/gcc/aarch64-redhat-linux/4.8.5
Candidate multilib: .;@m64
Selected multilib: .;@m64

**Comment by liuyong201811** [comment]
mpicc mpicxx mpif90


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "idaholab/moose"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月14日(星期天) 下午3:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [idaholab/moose] There is error when run  ./update_and_rebuild_petsc.sh (Discussion #19378)





 
@liuyong201811 what does this return?
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by liuyong201811** [reply]
mpicc --version
Bisheng Compiler 1.3.3.b023 clang version 10.0.1 (clang-e31092d4f8cd flang-3c92ea4b404f)
Target: aarch64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/compiler/bisheng-compiler-1.3.3-aarch64-linux/bin

**Comment by loganharbour** [comment, marked as answer]
Which of our instruction pages are you using to setup your build environment?

**Reply by liuyong201811** [reply]
We want to install Moose in  ARM  machine and clang flang clang++, Can you give some advice to us, thank you!

**Reply by milljm** [reply]
It is a straight forward process building on a Linux ARM machine (@dschwen used a Raspberry Pi3 as I recall, and I was on a Beaglebone, and one other successful attempt on a raspberry 4), but that was while using gcc, g++ and gfortran. Not the LLVM toolchain.

Flang is a fairly new compiler. I don't think anyone our team will have much if any experience. Indeed, when I search the internet for BLAS and Flang, I don't get promising hits. I am genuinely curious if this works. It would be nice to have an entirely LLVM solution.

**Comment by ljeft** [comment]
I installed moose in wsl for download and had the same problem when building PETSc.This is a problem I have not encountered before installing moose, I do not know how to solve it.

```pre
Mon Jan 29 21:29:55 CST 2024
No LSB modules are available.

System Arch: Distributor ID: Ubuntu Description: Ubuntu 22.04.3 LTS Release: 22.04 Codename: jammy

MOOSE Package Version: Custom Build

CPU Count: 28

Memory Free: 6164.730 MB

Variable `which $CC` check:
/usr/local/openmpi-4.1.6/bin/mpicc

$CC --version:
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

MPICC:
which mpicc:
        /usr/local/openmpi-4.1.6/bin/mpicc
mpicc -show:
        gcc -I/usr/local/openmpi-4.1.6/include -L/usr/local/openmpi-4.1.6/lib -Wl,-rpath -Wl,/usr/local/openmpi-4.1.6/lib -Wl,--enable-new-dtags -lmpi

COMPILER gcc:
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Python: NO PYTHON DETECTED

MODULES NOT AVAILABLE

PETSC_DIR not set

ENVIRONMENT:
CC=mpicc
CUDA_HOME=:/usr/local/cuda-11.8
CXX=mpicxx
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
DISPLAY=:0
F77=mpif77
F90=mpif90
FC=mpif90
HOME=/home/sohar
HOSTTYPE=x86_64
LANG=C.UTF-8
LD_LIBRARY_PATH=:/usr/local/cuda-11.8/lib64
LESSCLOSE=/usr/bin/lesspipe %s %s
LESSOPEN=| /usr/bin/lesspipe %s
LOGNAME=sohar
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
METHODS=opt
MOOSE_JOBS=6
NAME=DESKTOP-RPC0OMG
OLDPWD=/home/sohar
PATH=/usr/local/openmpi-4.1.6/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Program Files/WindowsApps/CanonicalGroupLimited.Ubuntu22.04LTS_2204.3.63.0_x64__79rhkp1fndgsc:/mnt/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/bin:/mnt/c/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v11.8/libnvvp:/mnt/c/Windows/system32:/mnt/c/Windows:/mnt/c/Windows/System32/Wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0/:/mnt/c/Windows/System32/OpenSSH/:/mnt/c/Program Files (x86)/NVIDIA Corporation/PhysX/Common:/mnt/c/Program Files/NVIDIA Corporation/NVIDIA NvDLISR:/mnt/c/Program Files/Microsoft SQL Server/150/Tools/Binn/:/mnt/c/Program Files/Microsoft SQL Server/Client SDK/ODBC/170/Tools/Binn/:/mnt/e/Windows Kits/10/Windows Performance Toolkit/:/mnt/c/Program Files/NVIDIA Corporation/Nsight Compute 2022.3.0/:/mnt/e/Git/cmd:/mnt/c/Users/DZKD/AppData/Local/Microsoft/WindowsApps:/mnt/d/Users/DZKD/AppData/Local/Programs/Microsoft VS Code/bin:/snap/bin:/usr/local/cuda-11.8/bin
PULSE_SERVER=unix:/mnt/wslg/PulseServer
PWD=/home/sohar/projects/moose/scripts
SHELL=/bin/bash
SHLVL=1
TERM=xterm-256color
USER=sohar
WAYLAND_DISPLAY=wayland-0
WSL2_GUI_APPS_ENABLED=1
WSLENV=
WSL_DISTRO_NAME=Ubuntu-22.04
WSL_INTEROP=/run/WSL/656_interop
XDG_DATA_DIRS=/usr/local/share:/usr/share:/var/lib/snapd/desktop
XDG_RUNTIME_DIR=/run/user/1000/
_=/usr/bin/env
```

**Reply by milljm** [reply]
Please create a new Discussion thread. Your issue is not the same as the original posters running on a different architecture (aarch64).

In the new Discussion please also include the actual error you are receiving, along with the above diagnostics output.

Thanks!
